From: Lucas Stach <l.stach@pengutronix.de>
Date: Fri, 11 May 2018 15:21:26 +0200
Subject: [PATCH] unconditionally add FStar to Makefile

This unit provides helper functions that are used by multiple other
units. Fixes freebl lib build with old build system.

Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
---
 nss/lib/freebl/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/nss/lib/freebl/Makefile b/nss/lib/freebl/Makefile
index e6a94d35c45a..58c75da95dba 100644
--- a/nss/lib/freebl/Makefile
+++ b/nss/lib/freebl/Makefile
@@ -535,12 +535,13 @@ ifeq (,$(filter-out i386 x386 x86 x86_64 aarch64,$(CPU_ARCH)))
     # All intel architectures get the 64 bit version
     # With custom uint128 if necessary (faster than generic 32 bit version).
     ECL_SRCS += curve25519_64.c
-    VERIFIED_SRCS += Hacl_Curve25519.c FStar.c
+    VERIFIED_SRCS += Hacl_Curve25519.c
 else
     # All non intel architectures get the generic 32 bit implementation (slow!)
     ECL_SRCS += curve25519_32.c
 endif
 
+    EXTRA_SRCS += FStar.c
 #######################################################################
 # (5) Execute "global" rules. (OPTIONAL)                              #
 #######################################################################
